python/tests/op_mappers/test_clip_op.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2023 CINN Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle
+import unittest
+from op_mapper_test import OpMapperTest
+
+
+class TestClipOp(OpMapperTest):
+    def init_input_data(self):
+        self.feed_data = {
+            'x': self.random([2], "float32"),
```

Comment:
min是-0.2的话，这儿应该加上min和max参数：
```suggestion
            'x': self.random([2], "float32", -1.0, 1.0),
```

Reply:
Done